src/app/public/modules/tabs/tabset.component.ts

```diff
@@ -171,12 +187,16 @@ export class SkyTabsetComponent
 
   public selectTab(tab: SkyTabComponent): void {
     if (this.permalinkId && tab.permalinkValue) {
-      return;
+      this.setQueryParamPermalinkValue(tab.permalinkValue);
```

Comment:
Should this function now be `setPathParamsPermalinkValue` since we are moving from query to path params?

Reply:
Thanks!